src/catalog/src/durable/persist.rs

```diff
@@ -1049,6 +1050,22 @@ impl OpenableDurableCatalogState for UnopenedPersistCatalogState {
             .ok_or(CatalogError::Durable(DurableCatalogError::Uninitialized))
     }
 
+    #[mz_ore::instrument(level = "debug")]
+    async fn get_enable_0dt_deployment(&mut self) -> Result<Option<bool>, CatalogError> {
+        let value = self.get_current_config(ENABLE_0DT_DEPLOYMENT).await?;
+        match value {
+            None => Ok(None),
+            Some(0) => Ok(Some(false)),
+            Some(1) => Ok(Some(true)),
```

Comment:
Is there some `bool::try_from(u: u64)` that we can use here? Not a big deal because I'm pretty sure this is the right mapping.

Reply:
Not that I could find!